packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -90,6 +90,11 @@ abstract class InAppPurchaseConnection {
   ///
   /// See the [example readme](../../../../example/README.md) for steps on how
   /// to initialize products on both payment platforms.
+  ///
```

Comment:
Is this the best place for this doc? I'd imaging most of the APIs won't work. I think specifying this in README is good enough.

Reply:
OKay will remove that.